.github/workflows/ci.yml

```diff
@@ -25,9 +25,21 @@ jobs:
           mix local.rebar --force
           mix local.hex --force
           mix deps.get
-      - name: Compile Code
-        run: mix compile --warnings-as-errors
       - name: Ensure Code is Formatted
         run: mix format --check-formatted
+      - name: Compile for Tests
+        run: MIX_ENV=test mix compile --warnings-as-errors
       - name: Run Tests
         run: mix test
+      - name: Generate Release
+        if: github.ref == 'refs/heads/master'
+        run: mix release && cp _build/prod/rel/bakeware/fika fika
+      - name: Upload Artifacts
+        if: github.ref == 'refs/heads/master'
+        uses: "marvinpinto/action-automatic-releases@latest"
+        with:
+          repo_token: "${{ secrets.GITHUB_TOKEN }}"
+          automatic_release_tag: "latest"
+          prerelease: true
+          title: "Development Build"
```

Comment:
@emilsoman any suggestions for the release title?

Reply:
This works :)